typescript/sdk/src/index.ts

```diff
@@ -314,6 +303,17 @@ export {
   defaultViemProviderBuilder,
   protocolToDefaultProviderBuilder,
 } from './providers/providerBuilders.js';
+export { TxSubmitterInterface } from './providers/transactions/submitter/TxSubmitterInterface.js';
```

Comment:
I don't know why different devs seem to get different results for the ordering here.